frontend/beCompliant/src/components/table/AnswerCell.tsx

```diff
@@ -1,15 +1,18 @@
-import { Button, Input, Select, Stack, Textarea } from '@kvib/react';
+import { Box, Button, Input, Select, Stack, Textarea, Text } from '@kvib/react';
 import { useState } from 'react';
 import { useParams } from 'react-router-dom';
 import { useSubmitAnswers } from '../../hooks/useSubmitAnswers';
 import { AnswerType } from '../../api/types';
+import { formatDateTime } from '../../utils/formatTime';
+import { LastUpdated } from './LastUpdated';
 
 type Props = {
   value: any;
   answerType: AnswerType;
   questionId: string;
   questionName: string;
   comment: string;
+  updated: Date;
```

Comment:
Er det altid en date eller kan den vare undefined når man ikke har kommentert?

Reply:
Du gjører en sjekk på linje 105 om den er undefined. så skal sikkert vare 
```suggestion
  updated?: Date;
```